ndcube/tests/test_global_coords.py

```diff
@@ -0,0 +1,21 @@
+import astropy.units as u
+import pytest
+
+from ndcube.global_coords import GlobalCoords
+
+
+@pytest.fixture
+def global_coords(ndcube_3d_ln_lt_l):
+    return GlobalCoords(ndcube_3d_ln_lt_l)
+
+
+def test_some_basic_coordinates(global_coords):
+    coord1 = 1 * u.m
+    coord2 = 2 * u.s
+    global_coords.add('name1', 'physical_type1', coord1)
+    global_coords.add('name2', 'physical_type2', coord2)
+    assert global_coords.names == ('name1', 'name2')
+    assert global_coords.physical_types == ('physical_type1', 'physical_type2')
+
+    global_coords.remove('name2')
+    assert len(global_coords) == 1
```

Comment:
It would be good to add some extra tests for iterating, slicing and the output of keys and values here as well.

Reply:
Let me give it a think and add extra tests for it

Reply:
Done

Reply:
Could you break out this test so that it tests one functionality at a time?  Namely, `add`, `remove` and `len`.  So break it into 3 tests.

Reply:
Done, will add `__len__()` test soon

Reply:
Added a simple `len` test for the case of `length=2`